tests/e2e/device/e2e_device_commands.c

```diff
@@ -780,6 +785,150 @@ static uint32_t prvE2ETestVerifyDesiredPropertiesCommandExecute( E2E_TEST_COMMAN
 }
 /*-----------------------------------------------------------*/
 
+/**
+ * Execute Writeable property response command

```

Comment:
This is more of a nit I think with this whole file (including the file name I guess). The `_commands` used in various places now has a different connotation with PnP being a thing. Could we look to change this to something else: maybe `_task` or `_job`? When I read this and the subsequent function call, it seems to be mixing properties and commands.

Reply:
This change will increase the scope of this PR.. I think we should take care of this in separate PR.

Reply:
yup definitely didn't want it part of this one